services/workflows-service/scripts/alerts/generate-alerts.ts

```diff
@@ -236,33 +225,36 @@ export const generateFakeAlertDefinition = async (
     );
 
     const createdBy = faker.internet.userName();
+
+    const ruleIdIdx = faker.datatype.number({
+      min: 0,
+      max: rules.length - 1,
+    });
```

Comment:
If we are still using `faker` I don't understand why not use the random array element like before?